### DISCUSSION TITLE
how to get the current plugin id in plugin's access phase?

### DISCUSSION THREAD

**Discussion by solo-xwz** [Kong/kong#9878] [Help] [answered]
how to get the current plugin id in plugin's access phase?

**Comment by samugi** [comment, marked as answer]
Hi @solo-xwz, what is the use case of obtaining the plugin id? If you need a key that uniquely identifies the plugin instance, you could use `conf.__key__`, which consists of a string including the id of whatever object the plugin is configured on, concatenated with the workspace id. Currently the plugin id is not available as an easily accessible field via conf.

**Reply by solo-xwz** [reply]
ok,thank you ,it seems that conf.__key__ can be used as well.i just want use it as a dict's key  in my custom plugin.
